test/endtoend/transloadit/test.js

```diff
@@ -0,0 +1,42 @@
+/* global browser, expect, capabilities, $ */
+const path = require('path')
+const { selectFakeFile, supportsChooseFile } = require('../utils')
+
+const testURL = 'http://localhost:4567/transloadit'
+
+function unhideTheInput () {
+  var input = document.querySelector('#uppy-transloadit .uppy-Dashboard-input')
+  input.removeAttribute('hidden')
+  input.removeAttribute('aria-hidden')
+  input.removeAttribute('tabindex')
+}
+
+function setTransloaditKeyAndInit (transloaditKey) {
+  window.initUppyTransloadit(transloaditKey)
+}
+
+describe('Transloadit file processing', () => {
+  beforeEach(() => {
+    browser.url(testURL)
+    browser.execute(setTransloaditKeyAndInit, process.env.TRANSLOADIT_KEY)
```

Comment:
We could skip this test if TRANSLOADIT_KEY is not available, so users don't have to set it up to run integration tests when they aren't actually changing anything Transloadit related. See eg the provider tests:

https://github.com/transloadit/uppy/blob/007504cefc12289cc6d4173b5fe6483ff03c1b53/test/endtoend/providers/test.js#L11-L14

I think we could do the check inside a `before(function () {})` section, calling `this.skip()` there should skip the entire test.

Reply:
Very good idea, thank you, added 👍